src/platform_impl/web/window.rs

```diff
@@ -268,6 +270,11 @@ impl Window {
         // Currently a no-op as it does not seem there is good support for this on web
     }
 
+    #[inline]
+    pub fn request_user_attention(&self, _request_type: Option<RequestUserAttentionType>) {
+        // Currently an intentional no-op
```

Comment:
warning ?

Reply:
I think it's done this way due to consistency with the backend other handlings of such things? I don't see a reason to diverge from existing backend code style...

Reply:
Jup, I stayed consistent with the different backends, which might look a bit weird in this PR but overall is probably better.